addons/web/static/src/webclient/navbar/navbar.xml

```diff
@@ -150,9 +152,9 @@
       </t>
       <t t-foreach="sections" t-as="section" t-key="section.id">
 
-        <t t-if="!section.childrenTree.length">
+        <t t-if="!section.childrenTree.length &amp;&amp; (!section.technical_chm || section.technical_chm == this.technical_chm.active)">
```

Comment:
t-if="!section.childrenTree.length and (!section.technical_chm || this.technical_chm.active)"